doc/source/python_reference.rst

```diff
@@ -76,12 +76,23 @@ Functions:
    :toctree: generated/
    :template: function.rst
 
-   read_hdf5
    read_tab
    read_wav
-   write_hdf5
```

Comment:
I think we still want these functions in this namespace. They should just call the underlying `h5io` version with the proper `expyfun` title argument.